packages/react/src/components/FloatingPortal.tsx

```diff
@@ -55,13 +62,18 @@ export const useFloatingPortalNode = ({
       }
       newPortalEl.setAttribute('data-floating-ui-portal', '');
       setPortalEl(newPortalEl);
-      const container = portalContext?.portalNode || document.body;
+
+      const container =
+        portalContext?.portalNode ??
+        floatingPortalRootNodeContext ??
+        document.body;
```

Comment:
(Just to reduce transpilation cost)
```suggestion
      const container =
        portalContext?.portalNode ||
        floatingPortalRootNodeContext ||
        document.body;
```